ha/core/event_analyzer/parser/parser.py

```diff
@@ -145,3 +152,60 @@ def parse_event(self, msg: str) -> HealthEvent:
 
         except Exception as e:
             raise EventParserException(f"Failed to parse IEM. Message: {msg}, Error: {e}")
+
+
+class ClusterResourceParser(Parser):
+    """
+    Subscriber for event analyzer to pass msg.
+    """
+
+    def __init__(self):
+        """
+        Init method.
+        """
+        super(ClusterResourceParser, self).__init__()
+        ConfigManager.init("event_analyzer")
+        Log.info("ClusterResource Parser is initialized ...")
+
+    def parse_event(self, msg: str) -> HealthEvent:
+        """
+        Parse event.
+        Args:
+            msg (str): Msg
+        """
+        try:
+            message = json.dumps(ast.literal_eval(msg))
+            cluster_resource_alert = json.loads(message)
+            cluster_id = Conf.get(const.HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}cluster_id")
+            site_id = Conf.get(const.HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}site_id")
+            rack_id = Conf.get(const.HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}rack_id")
+            timestamp = str(int(time.time()))
+            event_id = timestamp + str(uuid.uuid4().hex)
+            node_id = cluster_resource_alert["_resource_name"]
+            resource_type = cluster_resource_alert["_resource_type"]
+            event_type = cluster_resource_alert["_event_type"]
+            timestamp = cluster_resource_alert["_timestamp"]
+
+            event = {
+                EVENT_ATTRIBUTES.EVENT_ID : event_id,
+                EVENT_ATTRIBUTES.EVENT_TYPE : event_type,
+                EVENT_ATTRIBUTES.SEVERITY : StatusMapper.EVENT_TO_SEVERITY_MAPPING[event_type],
+                EVENT_ATTRIBUTES.SITE_ID : site_id, # TODO: Should be fetched from confstore
+                EVENT_ATTRIBUTES.RACK_ID : rack_id, # TODO: Should be fetched from confstore
+                EVENT_ATTRIBUTES.CLUSTER_ID : cluster_id, # TODO: Should be fetched from confstore
+                EVENT_ATTRIBUTES.STORAGESET_ID : node_id,
+                EVENT_ATTRIBUTES.NODE_ID : node_id,
+                EVENT_ATTRIBUTES.HOST_ID : node_id,
+                EVENT_ATTRIBUTES.RESOURCE_TYPE : resource_type,
+                EVENT_ATTRIBUTES.TIMESTAMP : timestamp,
+                EVENT_ATTRIBUTES.RESOURCE_ID : node_id,
+                EVENT_ATTRIBUTES.SPECIFIC_INFO : "specific_info"
+            }
+
+            Log.debug(f"Parsed {event} schema")
+            health_event = HealthEvent.dict_to_object(event)
+            Log.info(f"Event {event[EVENT_ATTRIBUTES.EVENT_ID]} is parsed and converted to object.")
```

Comment:
This should be log.debug so as not to flood the log

Reply:
Done